sql/core/src/main/java/org/apache/spark/sql/execution/datasources/parquet/VectorizedParquetRecordReader.java

```diff
@@ -39,12 +42,13 @@
 import org.apache.spark.sql.vectorized.ColumnarBatch;
 import org.apache.spark.sql.types.StructField;
 import org.apache.spark.sql.types.StructType;
+import scala.collection.JavaConverters;
```

Comment:
Is this import required?

Reply:
Yes it's used below in `checkColumn` method for example

Reply:
That should probably be moved after java imports 🙂. I am probably nit-picking here but maybe it would be a good nit to fix - it is up to you.